VideoEditing/VideoEditor/VideoEditor/Model/Configuration/VideoBuilderConfiguration.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  VideoBuilderConfiguration.swift
+//  VideoEditing
+//
+//  Created by Roland Huhn on 26/01/17.
+//  Copyright © 2017 3PillarGlobal. All rights reserved.
+//
+
+import UIKit
+
+class Resolution{
+    var width: Int
+    var height : Int
+    
+    init(width: Int, height: Int) {
+        self.width = width
+        self.height = height
+    }
+}
+
+enum VideoSize{
+    case HD
+    case FullHD
+    case UHD4K
+    
+    func values() -> (widht: Int, height: Int) {
```

Comment:
should we go with the naming from CGRect and rename the method to size? Should we use a function or a variable to return to width and height?